File: Lua/src/apuntes/variables.lua
-- Creación de variables
local palabra = "Candela"
local numero = 3.14
local booleano = true


-- Salida por pantalla
print("Hello World!") 
io.write("Hello World!")


-- Salida por pantalla de variables
local nombre = "Candela"
print("Hello ", nombre)


-- O también
local nombre = "Candela"
io.write("Hello ", nombre)


-- Ingreso por teclado de palabras
print("Ingrese su nombre: \n")
local nombre = io.read("*line")
print("Nombre: ", nombre)


-- Ingreso por teclado de números
print("Ingrese un numero: \n")
local numero = io.read("*n")
print("Numero: ", numero)

File: Java/src/apuntes/variables.java
import java.util.*;

class Main
{  
  public static void main(String args[])
  {  
    // Declaración de variables
    char letra = "c";
    String nombre = "Candela";
    int numeroEntero = 587;
    double numeroDecimal = 3.14;
    boolean booleano = true;


    // Salida por pantalla 
    System.out.println("Hello World!"); 

    String variable = "Candela";
    System.out.println("Hello " + variable);


    // Ingreso por teclado
    // Palabras:
    Scanner scan = new Scanner(System.in);
    String variable = scan.nextLine();
    scan.close();

    // Letras:
    Scanner scan = new Scanner(System.in);
    char variable = scan. next().charAt(0);
    scan.close();

    // Números enteros:
    Scanner scan = new Scanner(System.in);
    int variable = scan. nextInt();
    scan.close();

    // Números decimales:
    Scanner scan = new Scanner(System.in);
    double variable = scan. nextDouble();
    scan.close();

  }
}